modules/nodes-nodes-managing-max-pods-proc.adoc

```diff
@@ -43,8 +43,8 @@ metadata:
 .. If the label is not present, add a key/value pair:
 +
 ----
-oc label machineconfigpool worker custom-kubelet=small-pods
----- 
+# oc label machineconfigpool worker custom-kubelet=small-pods
```

Comment:
Do you need to be root to run the commands in this module?

Reply:
I doubt it, but the previous CLI commands in the file all used '#' and I wanted to be consistent. I didn't want to change those just in case it was actually purposeful and I just didn't realize why.

Reply:
That's fair. It's a better question for @mburke5678  for later, then. Our convention is to use # if you need root and $ otherwise. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks